plugins/command-line/prism-command-line.js

```diff
@@ -4,6 +4,67 @@ if (typeof self === 'undefined' || !self.Prism || !self.document) {
 	return;
 }
 
+// Scoped to this IIFE so they can be accessed by both hooks. -- cwells
+var outputLines = [];
```

Comment:
I had the same thought and considered sending a note to ask about adding a way to compartmentalize variables in the `env` object, but I didn't want to rock the boat too much for my own needs.  I would have done a few things differently if there was an obvious way to set variables without concern for stepping on anyone else's code.

I suggest adding a property on the `env` object specifically for plugin usage such as:

```javascript
env.pluginVars["command-line"].outputLines = [];
```

With this approach, every plugin (being uniquely named) could have its own variables in the `env.pluginVars` array using any names the developer chooses without regard to other plugins' variables.  Thoughts?